src/openrct2/actions/MazeSetTrackAction.hpp

```diff
@@ -260,11 +254,10 @@ DEFINE_GAME_ACTION(MazeSetTrackAction, GAME_COMMAND_SET_MAZE_TRACK, GameActionRe
                     uint8_t temp_edx = byte_993CFC[segmentOffset];
                     if (temp_edx != 0xFF)
                     {
-                        uint16_t previousElementX = floor2(_loc.x, 32) - CoordsDirectionDelta[_loc.direction].x;
-                        uint16_t previousElementY = floor2(_loc.y, 32) - CoordsDirectionDelta[_loc.direction].y;
+                        auto previousElement = CoordsXY{ _loc }.ToTileStart() - CoordsDirectionDelta[_loc.direction];
```

Comment:
Could you get location of some sort into this name. `Element` is almost always used to mean `TileElement` and it might be slightly confusing.

Reply:
Ok, renamed it to `previousElementLoc`